app/components/sidebars/main/teams_list/teams_list_item/teams_list_item.js

```diff
@@ -70,15 +77,26 @@ export default class TeamsListItem extends React.PureComponent {
             );
         }
 
+        const itemTestID = `${testID}.${teamId}`;
+        const displayNameTestID = `${testID}.display_name`;
+        const teamIconTestID = `${testID}.team_icon`;
```

Comment:
```suggestion
        const displayNameTestID = `${itemTestID}.display_name`;
        const teamIconTestID = `${itemTestID}.team_icon`;
```

right ?

Reply:
`${testID}.display_name` and `${testID}.team_icon` are intentional so that the component can be queried either by specific testID ancestor (using `withAncestor`) or by order using atIndex. This way, test cases that require either specific targeting of component or order of component can be implemented.